Project ready for marking

DarionRichards opened the pull request.

DarionRichards committed:
Update README

DarionRichards committed:
Added file structure

DarionRichards committed:
Basic structure of index.html

- Placed main elements used for semantic HTML.
- Added comments for sections of page

DarionRichards committed:
Added code to reset.css

DarionRichards committed:
Linked stylesheets

DarionRichards committed:
Added section for header

Added section for header which will contain:
- Logo
- Navigation Bar

DarionRichards committed:
Added styling for header section

- Created font family for body section
- Created classes for header section

DarionRichards committed:
Added Navigation Bar

- Created basic structure of navigation bar
- Added classes for styling
- Added IDs for referencing script for navigation bar

DarionRichards committed:
Added CSS styles

- Created :root colour scheme for website
- Created typography of web page, declared within body
- Added styles for nav bar
- Added media queries for Desktop/Laptop, Table and Mobile view

DarionRichards committed:
Added Script for dropdown feature in Navigation Bar

- Created event listener to run function when "clicked"
- Created a function, in order to list a class called "active" on a element with an identified ID
- For each ID, CSS will then declare the styling with the elementID.active

DarionRichards committed:
Added footer section to index.html

- Created container for styles.css
- Added section for social media information
- Created anchor tags for links to social media
- Added icons to footer

DarionRichards committed:
Added images for page

- Created file in assets

DarionRichards committed:
Added styles for footer section

- Created footer container
- Added CSS attributes to <p> element used within the container
- Added icon size for social media icons

DarionRichards committed:
Added basic structure to main section

DarionRichards committed:
Added headings for sections of page

index.html
- Created sections, with comments describing each section and their purpose

styles.css
- Created container class for seperating sections on the page
- Created heading class for highligting the different sections of the page (about me, projects, contact)

DarionRichards committed:
Centred content on page more

Adding padding to containers to contain information more center in the page
- Adjusted padding within navigation bar to match containers on page

DarionRichards committed:
Adjusted structure of styles.css

Adjusted structure of css so that media queries appear at the bottom of file.

DarionRichards committed:
Positioning of projects

Index.html
- Created div sections for each individual project, for styling and positioning
- Declared .flex-row class within projects container to declare two rows, containing 3 project on each row

styles.css
- Created selector called .flex-row
- Created .flex-row properties in order use the flex-direction of row

DarionRichards committed:
Added placeholder images

index.html
- Created <a> tags for links to projects
- Created <img> tags for placeholders

styles.css
- Added padding to space projects to concordantly seperate

DarionRichards committed:
Added titles to projects

DarionRichards committed:
Adjusted positioning in about section

- Created classes for title and text
- Added styling to classes used for title and text within About section

DarionRichards committed:
Fixed flexing of project cards

- Added margin to seperate contents

DarionRichards committed:
Added opacity properties to images

DarionRichards committed:
Updated README

DarionRichards committed:
Added alt attributes to <img> tags

DarionRichards committed:
Fixed navbar to top of page

index.html
- Declared class within <nav> to fix navigation bar to top of the web page
- Added id references to navigation links to scroll down page automatically

styles.css
- Had problem where navbar had a margin at the top, making it appear lower
- Soloution: declared top attribute to postion on page

DarionRichards committed:
Fixed styling of navigation bar

DarionRichards committed:
Added link to Horiseon Project GitHub page

DarionRichards committed:
Fixed footer background attribute

Wrong CSS variable used in background attribute to colour footer

DarionRichards committed:
Added comments to styles.css

DarionRichards committed:
Added links to social icons

DarionRichards committed:
Updated README

tashamann commented on assets/css/styles.css:
```diff
@@ -0,0 +1,376 @@
+
+/* dropdown list styling for navigation buttons*/
+
+#navItem1.active {
```
I think you mean to use the pseudo class `:active` here, rather than `.active` which would signal a class selector (same with the next group too)

tashamann commented on assets/css/styles.css:
```diff
@@ -0,0 +1,376 @@
+.main-project {
+    display: flex;
+    justify-content: center;
+    padding: 25px 0 25px 0;
```
Can simplify this further and use the shorthand for the `padding` attribute - `padding: 25px 0`. The first value is for top and bottom, the second value is for left and right, so useful when you want to apply the same value to T/B L/R

tashamann commented on assets/css/styles.css:
```diff
@@ -0,0 +1,376 @@
+}
+
+.flex-row {
+    padding: 0 0 25px 0;
```
If you just want to alter padding/margin of just one side, instead of explicitly putting all other sides to 0, just use `padding-bottom` or `padding-left` for example.

tashamann commented on index.html:
```diff
@@ -0,0 +1,169 @@
+                    <section class="main-project">
+                        <section class="project-card">
+                            <div class="display-topleft">Horiseon Project (Hover)</div>
+                            <a href="https://github.com/DarionRichards/horiseon-semantic-refactoring"><img class="img-opacity" src="./assets/images/01-html-css-git-homework-demo.png" alt="Horiseon Project Demo Example" /> </a>
```
To address the issue of the main project image not resizing for smaller view ports, you can add a class to the `<img>` element so that you can target it directly and adjust the size. Try adding a `width` property.

DarionRichards committed:
Refactored code

- refactored code
- created a wrapper for content on page
- fixed image resizing
- refactoed mobile view ports for easier editing of web page

DarionRichards committed:
Refactored further

DarionRichards committed:
Fixed styling of project cards

DarionRichards committed:
Added margin to headers

DarionRichards commented:
Resolved comments
